### ISSUE TITLE
False positive for clean-import-paths with dot in the filename

### ISSUE THREAD

**Issue by webgefrickel** [sasstools/sass-lint#536]
When I try to import a file that has a dot in it's filename, the rule clean-import-paths rule throws a warning.

```scss
@import '../../node_modules/inuit-normalize/generic.normalize';
```

The name of the file is `_generic.normalize.scss` — so the clean import path above is correct, or am I missing something? Maybe the rule should just check for scss/sass file extensions?

**Comment by bgriffith** [comment]
Hey @webgefrickel - Thanks for the report, I've replicated and have a fix for it. The rule was checking for non css-files where as it now checks for .sass, .scss and blank.
